assets/js/modules/analytics/components/common/GA4PropertyNotice.js

```diff
@@ -32,18 +32,19 @@ import { __ } from '@wordpress/i18n';
 import SettingsNotice, { TYPE_INFO } from '../../../../components/SettingsNotice';
 import Link from '../../../../components/Link';
 
+const LearnMore = () => (
+	<Link
+		href="https://sitekit.withgoogle.com/documentation/ga4-analytics-property/"
+		external
+		inherit
+	>
+		{ __( 'Learn more here.', 'google-site-kit' ) }
```

Comment:
Based on Figma mock-ups it should be just `Learn more`:
```suggestion
		{ __( 'Learn more', 'google-site-kit' ) }
```

Reply:
I'm just following the IB :smile: